src/stories/Quiz/Hooks/index.tsx

```diff
@@ -105,7 +103,7 @@ export default function HooksTemplate(args) {
     }
 
     const { currentQuestion } = state.quiz;
-    const currentQuestionData = state.quiz.currentQuestion.next_question;
+    const currentQuestionData = state?.quiz?.currentQuestion?.next_question;
```

Comment:
this can be simplified:
```
    const { currentQuestion } = state.quiz;
    const currentQuestionData = state?.quiz?.currentQuestion?.next_question;
```

also... 
`state?.quiz?.` implies that it is possible for `state.quiz` to be undefined here....
but `const { currentQuestion } = state.quiz;` will throw an error if state.quiz is undefined

mayhaps we are using `?.` a bit too aggressively, sending mixed signals to devs who may be trying to interpret this code